insuree/models.py

```diff
@@ -70,6 +70,13 @@ class Meta:
         managed = True
         db_table = 'tblFamilyTypes'
 
+class IncomeLevels(models.Model):
+    id  = models.AutoField(db_column='IncomeLevelID', primary_key=True)
+    french_version = models.CharField(db_column='FrenchVersion', max_length=200, blank=True, null=True)
```

Comment:
If we want to integrate with MV then the fields names have to change as countries might use other languages. Maybe to use first_language and second_language. If you change the translation management to Lokalise, then you might need  only one language here to know what the id corresponds to.